pyexchange/leverjfutures.py

```diff
@@ -127,7 +127,7 @@ def __repr__(self):
 
     @staticmethod
     def from_our_list(pair, trade):
-        return Trade(trade_id=trade['executionid'],
+        return Trade(trade_id=trade['executionId'],
```

Comment:
More of a Quality of life change, but it might be worth subclassing the `Trade` object as a `LeverJFutureTrade; likewise with `Order`. An example of this can be seen in: https://github.com/makerdao/pyexchange/blob/master/pyexchange/dydx.py#L53. This would enable us to cut out ~100 lines of code, and have a single source of truth for these Classes.

Reply:
Well in this specific section Leverj spot is using different fields that Leverj futures exchange, but yes i tend to agree with you. Let's leave that for version 2.

Reply:
Fair point; sounds good.